kasa/discover.py

```diff
@@ -50,6 +52,8 @@ def connection_made(self, transport) -> None:
         sock = transport.get_extra_info("socket")
         sock.setsockopt(socket.SOL_SOCKET, socket.SO_BROADCAST, 1)
         sock.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
+        if self.interface is not None and len(self.interface) > 0:
```

Comment:
`if self.interface:` is sufficient here.

Reply:
Fixed in the last commit ("if self.interface is not None" is more pythonic style btw)

Reply:
Yes it indeed is :+1: this can be merged after the tests pass. Please run `tox -e lint` in the source directory, it will fix the formatting issues.

Reply:
Done